WcaOnRails/app/views/posts/_post.html.erb

```diff
@@ -22,6 +22,6 @@
     <div>
       <em>Announced by <%= post.author.name %> on <%= local_time(post.created_at) %></em>
     </div>
-    <%= post.body.html_safe %>
```

Comment:
In earlier versions of Rails you needed to explicitly sanitize user input using `h` and Rails' convention used to be `<%=h post.body %>` (can't find old docs right now, but at least this SO answer: http://stackoverflow.com/a/692926), so I simply used that convention for other helpers (e.g. date format). I can change it if you want.